cmd/dependencyverifier/dependencyverifier.go

```diff
@@ -104,6 +104,27 @@ func convertToMap(modStr string) ([]module, map[module][]module) {
 	return mainModulesList, modMap
 }
 
+func getShortestPath(mod module) (path []string) {
+	modWhyStr, err := runCommand("go", "mod", "why", "-vendor", "-m", mod.name)
```

Comment:
I have found `go mod why` does not give accurate answers... I don't think we can use this for determining whether we have a path to a package/module